x-pack/legacy/plugins/infra/public/pages/logs/log_entry_rate/page_results_content.tsx

```diff
@@ -196,7 +197,12 @@ export const LogEntryRateResultsContent: React.FunctionComponent = () => {
         </EuiFlexItem>
         <EuiFlexItem grow={false}>
           <EuiPanel paddingSize="l">
```

Comment:
```suggestion
           <EuiPanel paddingSize="m">
```
But I saw that the log rate panels also have large paddings - maybe we should change them together?